static/css/main.ie.css

```diff
@@ -1580,7 +1580,7 @@ textarea.success {
   display: inline;
   margin-right: 0;
   zoom: 1;
-  behavior: url('/static/vendor/box-sizing-polyfill/boxsizing.htc');
+  behavior: url('/src/vendor/box-sizing-polyfill/boxsizing.htc');
```

Comment:
Your change in `cf-theme-overrides.less` L139 results in this, which will not work. `src` will not be in the path when serving the site.